rig/type_casts.py

```diff
@@ -243,6 +243,50 @@ def __call__(self, values):
         return self.dtype(vals)
 
 
+class NumpyFixToFloatConverter(object):
+    """A callable which converts Numpy arrays of floats to fixed point arrays.
```

Comment:
"A callable which converts Numpy arrays of fixed-point values to floats." surely?